pkg/florist/netstat.go

```diff
@@ -113,3 +113,33 @@ func PublicIPs() ([]string, error) {
 
 	return ips, nil
 }
+
+// PrivateIP returns a first matching IP that belong to the given network CIDR.
+func PrivateIP(networkCIDR string) (string, error) {
+	_, network, err := net.ParseCIDR(networkCIDR)
+	if err != nil {
+		return "", fmt.Errorf("network CIDR: %s: error: %s", networkCIDR, err)
+	}
+
+	ips, err := PrivateIPs()
+	if err != nil {
+		return "", err
+	}
+
+	var match net.IP
+	for _, ip := range ips {
+		ip := net.ParseIP(ip)
+		if ip == nil {
+			continue
+		}
+
+		if network.Contains(ip) {
+			match = ip
+			break
+		}
+	}
+	if match == nil {
+		return "", fmt.Errorf("none of the private IPs belong to %s network", networkCIDR)
```

Comment:
belong -> belongs (the subject is "none" which is singular)

Sigh I thought it was so simple, then I read https://www.grammarbook.com/blog/subject-and-verb-agreement/none-plural-or-singular/ and I am now totally confused :-/

Reply:
I used plural as suggested :)